charts/newrelic-pixie/values.yaml

```
@@ -21,6 +21,8 @@ image:
   repository: newrelic/newrelic-pixie-integration
   tag: ""
   pullPolicy: IfNotPresent
+  # pullSecrets:
+  # - name: regsecret
```

Take a look at variables like these:
```yaml
proxy: {}
nodeSelector: {}
tolerations: []
affinity: {}
```
[Helm falsiness](https://helm.sh/docs/chart_template_guide/control_structures/#ifelse) is going to detect if as false if is empty. Better empty than commented :)

The comment as an example is perfect.
```suggestion
  pullSecrets: []
  # - name: regsecret
```